src/components/deploymentConfig/DeploymentConfig.tsx

```diff
@@ -755,7 +878,7 @@ export default function DeploymentConfig({
             className={`white-card__deployment-config p-0 bcn-0 ${state.openComparison ? 'comparison-view' : ''} ${
                 state.showReadme ? 'readme-view' : ''
             }`}
-            onSubmit={handleSubmit}
+            onSubmit={(e) => handleSaveChanges(e)}
```

Comment:
it can be written as `onSubmit={handleSaveChanges}`

Reply:
done